server/configs/application.properties

```diff
@@ -29,6 +29,9 @@ context.validationQuery[0]=SELECT 1
 
 #context.webAppLocation=@@pathToServer@@/build/deploy/labkeyWebapp
 context.encryptionKey=@@encryptionKey@@
+context.oldEncryptionKey=oldKey
+context.requiredModules=rM
+context.pipelineConfig=pC
```

Comment:
Should these be commented out by default?

Reply:
Yes, they should. Thanks. Now they are.